src/webloader.js

```diff
@@ -0,0 +1,126 @@
+import { generate } from './guids.js';
+import EvolvAssetManager from './index.js';
+import EvolvClient from "@evolv/javascript-sdk";
+
+
+function ensureId(evolv, key, session) {
+	let id;
+	if (evolv.context && key in evolv.context) {
+		id = evolv[key];
+	} else {
+		id = evolv.retrieve(key, session);
+		if (!id) {
+			id = generate();
+			evolv.store(key, id, session);
+		}
+	}
+	return id;
+}
+
+function currentScript() {
+	if (document.currentScript) {
+		return document.currentScript;
+	}
+
+	for (let i = 0; i < document.scripts.length; i++) {
+		const script = document.scripts[i];
+		if (script && script.dataset && 'env' in script.dataset) {
+			return script;
+		}
+	}
+}
+
+function injectScript(endpoint, env, uid) {
+	const script = document.createElement('script');
+	script.type = 'text/javascript';
+	script.src = endpoint + '/' + env + '/' + uid + '/assets.js';
+
+	document.head.appendChild(script);
+}
+
+function injectStylesheet(endpoint, env, uid) {
+	const stylesheet = document.createElement('link');
+	stylesheet.setAttribute('rel', 'stylesheet');
+	stylesheet.setAttribute('type', 'text/css');
+	stylesheet.setAttribute('href', endpoint + '/' + env + '/' + uid + '/assets.css');
+
+	document.head.appendChild(stylesheet);
+}
+
+function main() {
+	if (!window.evolv) {
+		window.evolv = {};
+	}
+
+	const evolv = window.evolv;
+
+	if (!evolv.store) {
+		evolv.store = function (key, value, session) {
+			(session ? window.sessionStorage : window.localStorage).setItem('evolv_' + key, value);
```

Comment:
Less important, but again for symmetry, the execution plan uses `evolv:` as prefix for storage values.
```suggestion
			(session ? window.sessionStorage : window.localStorage).setItem('evolv:' + key, value);
```